workspace/teamdata/generate_report.py

```diff
@@ -131,16 +160,16 @@ def get_project_cards(project_id):
 
 def get_slack_username(github_username):
     return {
-        "CarolineMorton": "@Caroline",
-        "Jongmassey": "@Jon",
-        "StevenMaude": "@Steve",
-        "evansd": "@dave",
-        "iaindillingham": "@Iain",
-        "inglesp": "@inglesp",
-        "milanwiedemann": "@Milan",
-        "rebkwok": "@Becky S",
-        "robinyjpark": "@Robin",
-    }.get(github_username) or github_username
+        "CarolineMorton": "<@Caroline>",
+        "Jongmassey": "<@Jon>",
+        "StevenMaude": "<@Steve>",
+        "evansd": "<@dave>",
+        "iaindillingham": "<@Iain>",
+        "inglesp": "<@inglesp>",
+        "milanwiedemann": "<@Milan>",
+        "rebkwok": "<@Becky S>",
+        "robinyjpark": "<@Robin>",
+    }.get(github_username) or f"<@{github_username}>"
```

Comment:
This assumes that a user's GH username matches their Slack username, which won't always hold.